### ISSUE TITLE
Re-patching an already patched installation is broken

### ISSUE THREAD

**Issue by generic-user** [sindrets/atom-title-bar-replacer#36]
Since last update I am unable to re-patch the app.asar file.

Anyone else having the same problem?

```
Uncaught Error: EBUSY: resource busy or locked, unlink 'C:\Users\[User Name]\AppData\Local\atom\app-1.27.1\resources\app-FRAMELESS.asar'
```

**Comment by sindrets** [comment]
@generic-user I just updated Atom and patched with the `window frame remover` without any problem. However I was able to reproduce your problem. The problem occurs when you have already patched and try to re-patch. For some reason `app-FRAMELESS.asar` is marked as busy and is therefore not deleted and the patch cannot proceed. However if you already have the `app-FRAMELESS.asar` file, it means you have already patched and you should follow my instructions to delete `app.asar` and replace it with the `app-FRAMELESS.asar`.
Just to be clear: The patcher is ***not*** broken on Atom 1.27, but re-patching an already patched installation is broken. But nobody will ever have to do that, and if for some reason you do, just delete `app-FRAMELESS.asar` manually while Atom is closed, before re-performing the patch.

**Comment by generic-user** [comment]
Hi, for some reason I replaced the `app.asar` with the `app.asar.bak`, and completely ignored the  `app-FRAMELESS.asar.`, so the error is indeed completely my own foolishness. Luckily my post wasn't completely useless, as u distilled another bug from it.

Thanks for investigating, I really appreciate the effort!

**Comment by sindrets** [comment]
Fixed in v2.0.0. The window frame remover is no longer needed. There's a new setting for hiding the native title bar.
